src/main/java/graphql/analysis/QueryTraverser.java

```diff
@@ -177,7 +177,7 @@ private List<Node> childrenOf(Node<?> node) {
 
```

Comment:
Update file to bring in line with v21 - it's because ConditionalNodes are referenced in the ExecutableNormalizedOperationFactory, and those changes are viral in a few other files